addons/wp/functions/fnc_taskReset.sqf

```diff
@@ -26,14 +29,8 @@ if (_group isEqualType objNull) then { _group = group _group; };
 // units
 private _units = units _group select {!isPlayer _x};
 
-// make new group + adopt name
-private _groupNew = createGroup [(side _group), true];
-_groupNew setGroupIdGlobal [groupId _group];
-_groupNew setFormation (formation _group);
-
 // remove all current waypoints
 [_group] call CBA_fnc_clearWaypoints;
```

Comment:
we should not remove Waypoints on Soft Reset

Reply:
I disagree. It isn't much of a reset if the unit continues along its previous orders!

Reply:
the issue with this function is that it destroys placed waypoints in the mission. and this function is getting called automatically. we should never do such a thing. i believe it is against your motto of not interfering with waypoints in the main part of lambs_danger!

Reply:
You are missing the point.  The "allow reinforcement" variable isn't automatically enabled. The mission maker _MUST_ enable it himself.  Reinforcements isn't something that just happens. No previously made missions will be meaningfully altered.  Furthermore the "taskReset" only triggers for units which have their PATH stopped.  From example our own waypoints or CBA_fnc_TaskDefend. 

Hence. A mission makers that enables BOTH our modules AND the "allow reinforcement" variable can't be said to not willfully be bringing this on himself.

Reply:
then we should add a warning somewhere that informs the users that using reinforcements may delete Waypoints.

Reply:
You might be right. Of course, when we do a hard reset, we make a new group so the concern regarding Waypoints also disappears. 

Suggested compromise:  We add another argument that defaults to false. If set to true, it removes waypoints. @jokoho48